test/ssl_old_test.c

```diff
@@ -310,6 +310,10 @@ static int cb_server_alpn(SSL *s, const unsigned char **out,
      * verify_alpn.
      */
     alpn_selected = OPENSSL_malloc(*outlen);
+    if (alpn_selected == NULL) {
+        OPENSSL_free(protos);
+        abort();
```

Comment:
If we're going to abort then an `fprintf` describing the problem might be nice like we do after `next_protos_parse` above

Reply:
Thanks, I have submitted a fixup to add the fprintf.